src/SlackConnector/Connections/Clients/Chat/FlurlChatClient.cs

```diff
@@ -13,13 +13,14 @@ internal class FlurlChatClient : IChatClient
     {
         private readonly IResponseVerifier _responseVerifier;
         internal const string SEND_MESSAGE_PATH = "/api/chat.postMessage";
+        internal const string DELETE_MESSAGE_PATH = "/api/chat.delete";
 
         public FlurlChatClient(IResponseVerifier responseVerifier)
         {
             _responseVerifier = responseVerifier;
         }
 
-        public async Task PostMessage(string slackKey, string channel, string text, IList<SlackAttachment> attachments)
+        public async Task<PostMessageResponse> PostMessage(string slackKey, string channel, string text, IList<SlackAttachment> attachments)
```

Comment:
Doh, I have a branch with SpecsFor removed but isn't ready for merging. For now in the say tests you can just replace the ".Wait" with a ".Result" to get the response. I will then pull it into the new branch later and update it.